### DISCUSSION TITLE
Extracted data in hugging face?

### DISCUSSION THREAD

**Discussion by brando90** [lean-dojo/LeanDojo#60] [Q&A] [answered]
Hi! I'd like to train on all the extracted data. 

Is this the right repo?

https://huggingface.co/datasets/tasksource/leandojo

Or is it better to unzip the Zenodo stuff and figure out how to upload that then train on it?

Thanks in advance!

**Comment by yangky11** [comment, marked as answer]
This data repo on Hugging Face is not created/maintained by us. You can use it at your own discretion, but only the versions on Zenodo are official.
